Cargo.lock

```diff
@@ -0,0 +1,267 @@
+# This file is automatically @generated by Cargo.
```

Comment:
will it conflict with tikv's Cargo.lock when building with tikv?

Reply:
No, rust resolve dependency conflicts by mangling. Also, a lot of the dependencies in Cargo.lock of TiKV is only used in `rust-rocksdb`, so if we update here, the dependencies in Cargo.lock of TiKV will also update when updating `rust-rocksdb`